bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -285,7 +285,7 @@ object SpProgramFactory {
     val category = new Gsa.Category(~proposal.category.map(_.value()))
     val pi       = gsaPhase1DataInvestigator(proposal.investigators.pi)
     val cois     = proposal.investigators.cois.map(gsaPhase1DataInvestigator).asJava
-    new Gsa(abstrakt, category, pi, cois)
+    new Gsa(abstrakt, category, java.util.Collections.emptyList[Keyword], pi, cois)
```

Comment:
Were we sending the keywords to the archive?

Reply:
There's a comment that indicates we were sending them to the archive along with categories but I can't find any evidence of sending either now.